### ISSUE TITLE
Missing docs on how to deploy rollback

### ISSUE THREAD

**Issue by mwleeds** [ostreedev/ostree#1808]
After running `ostree admin unlock --hotfix` you are left with two deployments, the unlocked one and the locked one, the latter of which is labeled "(rollback)" in the `ostree admin status` output. But it's unclear how to deploy the rollback one. The man page for `ostree admin deploy` says it takes a refspec but the only difference between the two deployments is the deployment index.

**Comment by jlebon** [comment]
Do you want to deploy a new copy of the rollback (new etc merge), or actually put back the rollback as the default? (Related: I forgot that `ostree admin` doesn't have `rollback` like in RPM-OSTree). The former is just `ostree admin deploy $REFSPEC`, right?

**Comment by mwleeds** [comment]
> Do you want to deploy a new copy of the rollback (new etc merge), or actually put back the rollback as the default? (Related: I forgot that `ostree admin` doesn't have `rollback` like in RPM-OSTree). The former is just `ostree admin deploy $REFSPEC`, right?

In this case the latter is what I wanted, to go back to the state before the hotfix. I suppose `ostree admin deploy $REFSPEC` would work as long as `$REFSPEC` still points to the same commit and that command doesn't check for updates. I would think there would be more direct way though.

**Comment by jlebon** [comment]
Yeah, seems like we're missing something analogous to `rpm-ostree rollback`.

**Comment by jlebon** [comment]
Related: #380. Though that one is in the context of automatic rollbacks.

**Comment by westurner** [comment]
Is there a way to do this without reboot

Use case: Create a deployment with `sudo ostree admin unlock --hotfix` and then remove it without rebooting

```sh
sudo ostree admin status | grep '(rollback)$'
```
